src/python/pants/option/global_options.py

```diff
@@ -76,7 +78,8 @@ def from_bootstrap_options(cls, bootstrap_options):
     remote_store_chunk_bytes=1024*1024,
     remote_store_chunk_upload_timeout_seconds=60,
     remote_store_rpc_retries=2,
-    process_execution_parallelism=multiprocessing.cpu_count()*2,
+    local_execution_parallelism=multiprocessing.cpu_count(),
```

Comment:
Nitpick: since we have so many of these their names should be namespaced probably.

So `process_execution_local_parallelism`, here and elsewhere. Sorry!